pkg/services/ngalert/metrics/remote_alertmanager.go

```diff
@@ -0,0 +1,29 @@
+package metrics
+
+import (
+	"github.com/grafana/dskit/instrument"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+type RemoteAlertmanager struct {
+	HTTPRequestDuration *instrument.HistogramCollector
+	LastReadinessCheck  prometheus.Gauge
+}
+
+func NewRemoteAlertmanagerMetrics(r prometheus.Registerer) *RemoteAlertmanager {
+	return &RemoteAlertmanager{
+		HTTPRequestDuration: instrument.NewHistogramCollector(promauto.With(r).NewHistogramVec(prometheus.HistogramOpts{
+			Namespace: Namespace,
+			Subsystem: Subsystem,
+			Name:      "remote_alertmanager_http_request_duration_seconds",
```

Comment:
nit: In my experience, `http_request_duration` is typically a histogram for API endpoints. The more common nomenclature for measuring client-side latency is `latency_seconds` such as:

1. `alertmanager_notification_latency_seconds_bucket`
2.  `prometheus_notifications_latency_seconds`

My suggestion is `grafana_alerting_remote_alertmanager_latency_seconds`, but I don't feel strongly.